src/WordMatcher.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * Base class for function reference fixers.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class WordMatcher
+{
+    /**
+     * @var string[]
+     */
+    private $dict;
```

Comment:
`$dict` is a bit confusing IMO, why not simply `$candidates`?

Reply:
both good, changing